proto/viam/app/v1/app.proto

```diff
@@ -858,6 +864,8 @@ message Module {
   // A list of models that are available in the module
   repeated Model models = 7;
   // The total number of robots using this module
+  // The executable to run to start the module program
+  string entrypoint = 11;
   int64 total_robot_usage = 8;
```

Comment:
[nit] This is cutting a comment/code pair in half

Reply:
Good catch, updated.